### PR TITLE
Find the glyphs in a .glyphs and sketch out wiring to process them

### PR DESCRIPTION
**Pull request by rsheeter**
For now don't do anything meaningful with them.

NOTE: setup to target something other than main. @cmyr thought that once the prior PR in the chain (#21) merges this might auto-redirect to main so... lets find out.

### CODE REVIEW

File: glyphs2fontir/src/source.rs [hunk 1]
```diff
@@ -168,4 +191,40 @@ mod tests {
...
+            "glyphname".to_string(),
+            Plist::String("space".to_string(), 0..1),
+        );
+        assert_eq!(
+            "space",
+            glyph_name(
+                Path::new("f.glyphs"),
+                &glyph,
+                &(1usize..12usize),
+                "I'm a plist"
```
Comment (anthrotype) [inline glyphs2fontir/src/source.rs:276]: wait, why is this "I'm a plist"? Shouldn't this be something like `"glyphname = space"`?

Comment (anthrotype) [reply glyphs2fontir/src/source.rs:276]: ah, it's unused if glyph["glyphname"] is a Plist::String.. still it's confusing

File: glyphs2fontir/src/source.rs [hunk 2]
```diff
@@ -27,15 +27,21 @@ impl GlyphsIrSource {
 }
 
 struct PlistCache {
-    _global_metadata: StateSet,
+    global_metadata: StateSet,
     _root_dict: HashMap<String, Plist>,
```
Comment (anthrotype) [inline glyphs2fontir/src/source.rs:31]: underscore prefix is unnecessary for a private-by-default struct field, no?

File: glyphs2fontir/src/source.rs [hunk 3]
```diff
@@ -94,21 +100,32 @@ impl GlyphsIrSource {
...
-        _root_dict: &HashMap<String, Plist>,
+        root_dict: &HashMap<String, Plist>,
+        raw_plist: &str,
     ) -> Result<StateSet, Error> {
-        todo!()
+        // Naive mk1: if anything other than glyphs and date changes do a global rebuild
+        // TODO experiment with actual glyphs saves to see what makes sense
+        let mut state = StateSet::new();
+        for (key, plist) in root_dict {
+            if key == "glyphs" || key == "date" {
```
Comment (anthrotype) [inline glyphs2fontir/src/source.rs:110]: this is probably too coarse / too sensitive to changes, but I'm ok to keep like that until we have a better idea of what can change globally that would/would not affect single glyphs IR.

File: glyphs2fontir/src/source.rs (hunk 1 above)
Comment (rsheeter) [reply glyphs2fontir/src/source.rs:276]: I will update to something clearer, e.g. "Not used by test"

File: glyphs2fontir/src/source.rs (hunk 2 above)
Comment (rsheeter) [reply glyphs2fontir/src/source.rs:31]: I would have sworn it was warning about it, let me double-check

File: glyphs2fontir/src/source.rs (hunk 3 above)
Comment (rsheeter) [reply glyphs2fontir/src/source.rs:110]: SGTM

**Review by anthrotype — APPROVED** [review]

File: glyphs2fontir/src/source.rs (hunk 2 above)
Comment (rsheeter) [reply glyphs2fontir/src/source.rs:31]: global_metadata is now used, I tried with root_dict and it did complain:

```
warning: field `root_dict` is never read
  --> glyphs2fontir/src/source.rs:31:5
   |
29 | struct PlistCache {
   |        ---------- field in this struct
30 |     global_metadata: StateSet,
31 |     root_dict: HashMap<String, Plist>,
   |     ^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default
```

**Pull request merged**